src/streams.h

```diff
@@ -61,6 +61,7 @@ class OverrideStream
 
     int GetVersion() const { return nVersion; }
     int GetType() const { return nType; }
+    size_t size() const { return stream->size(); }
```

Comment:
nit: could be nice to put `size` with `read`, `write`, as they are all delegated.